src/ontology/metadata/doid.metadata.sssom.yml

```diff
@@ -28,6 +29,8 @@ curie_map:
   SNOMEDCT_US_2020_09_01: http://identifiers.org/snomedct/
   SNOMEDCT_US_2021_07_31: http://identifiers.org/snomedct/
   SNOMEDCT_US_2021_09_01: http://identifiers.org/snomedct/
+  SNOMEDCT_US_2022_07_31: http://identifiers.org/snomedct/
+  SNOMEDCT_US_2022_10_31: http://identifiers.org/snomedct/
```

Comment:
I added these but also added a SPARQL query to remove these. @matentzn , do we need to make a decision on this now? It seems like every release will have a new prefix (since there is a date).

Reply:
The snomed ones I just want to require so they don't fall under the remove situation; but it is a bit annoying this :)